File: server/src/main/java/com/rnazarapps/server/model/Task.java
package com.rnazarapps.server.model;

import javax.persistence.*;
import java.time.LocalDateTime;
import java.util.List;

@Entity
public class Task {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String title;
    private String description;
    private LocalDateTime startTime;
    private LocalDateTime endTime;

    @ElementCollection
    private List<Long> executors;
    @ElementCollection
    private List<String> tags;

}

File: app/src/main/java/com/rnazarapps/commsandtaskmanageapp/model/Task.kt
package com.rnazarapps.commsandtaskmanageapp.model

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

// Клас для задач
data class Task(
    @SerializedName("id") val id: Long,
    @SerializedName("taskChannelId") val taskChannelId: Long,
    @SerializedName("startTime") val startTime: LocalDateTime?,
    @SerializedName("endTime") val endTime: LocalDateTime?,
    @SerializedName("executors") val executors: List<User>,
    @SerializedName("comments") val comments: List<Comment>,
    @SerializedName("tags") val tags: List<Tag>
)